user/account/src/test/java/in/projecteka/jataayu/user/account/viewmodel/UserAccountsViewModelTest.kt

```diff
@@ -93,6 +100,42 @@ class UserAccountsViewModelTest {
 //        Mockito.verify(getLinkedAccountsCall).enqueue(ArgumentMatchers.any())
 //        Assert.assertEquals(linkedAccountsResponse, viewModel.linkedAccountsResponse.value)
     }
+   // @Test
+//    fun shouldDisplayAccounts(){
+//
+//        val payloadLiveData = PayloadLiveData<LinkedAccountsResponse>()
+//        val linkedAccountsResponse = getLinkedAccountsData()
+//
+//        Mockito.`when`(repository.getUserAccounts()).thenReturn(payloadLiveData)
+//        Mockito.`when`(profileResponseCall.enqueue(ArgumentMatchers.any()))
+//            .then { invocation ->
+//                val callback = invocation.arguments[0] as Callback<LinkedAccountsResponse>
+//                callback.onResponse(profileResponseCall, Response.success(linkedAccountsResponse))
+//            }
+//        viewModel.getUserAccounts()
+//        Mockito.verify(repository).getUserAccounts()
+//        Mockito.verify(profileResponseCall).enqueue(ArgumentMatchers.any())
+//        Assert.assertEquals(linkedAccountsResponse, viewModel.linkedAccountsResponse.value)
+//    }
+
+//    @Test
+//    fun shouldGetAccountsToDisplay() {
```

Comment:
remove commented code. comment with "//TODO: Modify after refactor" if this test will be added in the future.

Reply:
all commented code and TODO are already removed in the latest commit. Can you check this 70f362a commit